src/en/tutorial-client-basic.md

```diff
@@ -0,0 +1,367 @@
+### Changing models
+
+To manually change to a different model, say the original 'default' model:
+
+```bash
+juju switch default
+```
+
+### Adding a machine
+
+To request that machine be created that is devoid of an application:
+
+```bash
+juju add-machine -m alpha -n 2
+```
+
+Here we've used the `-m` option to explicitly select a model (i.e. we don't
+want the currently active model, which at this time is 'default').
+
+### Listing machines
+
+To change context to model 'alpha' and then list the model's machines:
+
+```bash
+juju switch alpha
+juju machines
+```
+
+Our example:
+
+```no-highlight
+Machine  State    DNS            Inst id        Series  AZ  Message
+0        started  10.243.67.116  juju-ded876-0  bionic      Running
+1        started  10.243.67.53   juju-ded876-1  bionic      Running
+```
+
+### Deploying to a LXD container
+
+To deploy 'apache2' to a new LXD container on existing machine '0':
+
+```bash
+juju deploy apache2 --to lxd:0
+```
+
+It's not necessary to explicitly target an existing machine in order to deploy
+a charm upon on. To deploy 'mongodb' on the second available machine ('1'):
+
+```bash
+juju deploy mongodb
+```
+
+### Scaling up an application
+
+To scale out the 'apache2' application by creating one or more instantiations
+(*units*) of it on new machines:
+
+```bash
+juju add-unit -n 1 apache2
+```
+
+### Viewing the model status
+
+The `status` command is one that you'll use often. It gives live information
+for a given model.
+
+```bash
+juju status
+```
+
+Our example currently shows:
+
+```no-highlight
+Model  Controller  Cloud/Region         Version  SLA          Timestamp
+alpha  lxd         localhost/localhost  2.5.0    unsupported  21:28:33Z
+
+App      Version  Status   Scale  Charm    Store       Rev  OS      Notes
+apache2           unknown      2  apache2  jujucharms   26  ubuntu  
+mongodb  3.6.3    active       1  mongodb  jujucharms   52  ubuntu  
+
+Unit        Workload  Agent  Machine  Public address  Ports 				    Message
+apache2/0*  unknown   idle   0/lxd/0  10.72.88.191                                             
+apache2/1   unknown   idle   2        10.243.67.141                                            
+mongodb/0*  active    idle   1        10.243.67.53 27017/tcp,27019/tcp,27021/tcp,28017/tcp  Unit is ready
+
+Machine  State    DNS            Inst id              Series  AZ  Message
+0        started  10.243.67.116  juju-ded876-0        bionic      Running
+0/lxd/0  started  10.72.88.191   juju-ded876-0-lxd-0  bionic      Container started
+1        started  10.243.67.53   juju-ded876-1        bionic      Running
+2        started  10.243.67.141  juju-ded876-2        bionic      Running
+```
+
+The output is broken up into four sections.
+
+The **top** section mentions basic information such as the names of the current
+model and controller ('alpha' and 'lxd' respectively), followed by the cloud
+name ('localhost'), what version is running on the agents in the current model,
```

Comment:
"what version is running on the agents in the current model" I'd suggest "what version of Juju the model is running"

Reply:
I thought I'd make it clearer using agents since it's hard to imagine "Juju running in the model". That's an abstraction. Dunno which is more confusing.

Reply:
I'll go with the abstraction.